nimble/transport/socket/src/ble_hci_socket.c

```diff
@@ -281,6 +343,9 @@ ble_hci_sock_cmdevt_tx(uint8_t *hci_ev, uint8_t h4_type)
     memset(&msg, 0, sizeof(msg));
     memset(iov, 0, sizeof(iov));
 
+    msg.msg_name = (void *)btaddr;
+    msg.msg_namelen = 6;
+
```

Comment:
why do we need this here? there's separate code for nuttx below. if this fixes something else than nuttx, please create separate PR.

Reply:
Right. I initially tried to use sendmsg() for NuttX but that is not implemented yet for Bluetooth sockets, this was leftover code. I just removed it.